src/schema.toml

```diff
@@ -8,6 +8,7 @@ translations.en."app-short-domain" = { category = "translations", type = "string
 translations.en."contact-email" = { category = "translations", type = "string", name = "Contact email", description = "General contact email." }
 translations.en."share-hashtag" = { category = "translations", type = "string", name = "Share Hashtag", description = "Hashtag used in share messages." }
 
+features.permissive_rooms = { category = "features", type = "boolean", name = "Permissive Rooms", description = "Use permissive room settings for new rooms." }
```

Comment:
I'd change this to `Enable full content creation permissions on new rooms` or something similar?

Reply:
Gotcha. Would be nice if you could do a pass on the copy for the configs when the dust settles.